git-format-pkg-patch

```diff
@@ -235,7 +235,7 @@ def do_create_patches(args):
     patches = 0
     for fname in os.listdir(current_dir):
         if fname.split('.')[-1] == 'patch':
-            print "Preparing", fname
+            print("Preparing", fname)
```

Comment:
This generates a weird output:
```
...
('Preparing', '0113-Add-IP-filtering-by-network.patch')
...
```

I would suggest the following to get the clean message:
```suggestion
            print("Preparing {}".format(fname))
```

Reply:
That's weird. I agree that the suggestion is better, but I get this:

```
Preparing 0040-temporary-fix-extend-the-whitelist-of-allowed-comman.patch
Preparing 0128-Catch-NameError-in-IPCClient.close-231.patch
Preparing 0060-batch.py-avoid-exception-when-minion-does-not-respon.patch
Preparing 0004-avoid-excessive-syslogging-by-watchdog-cronjob-58.patch
```